keyboards/durgod/dgk6x/config.h

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2021 Jessica Sullivan and Don Kjer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xD60D
+#define MANUFACTURER    Hoksi Technology
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION ROW2COL
+
+// Dynamic EEPROM
+// Something sensible or else VIA may crash
+// Users may enable more if they wish
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR  4095
+
+// Increase VIA layer count
+#define DYNAMIC_KEYMAP_LAYER_COUNT 16
```

Comment:
Oof.  This may cause eeprom init to take FOREVER.  8 may be a more reasonable value.

Reply:
I haven't noticed any issues or slowdown with 16 layers

Reply:
Could you elaborate? I have tested and didn't notice any issues.

Reply:
When VIA first loads the keymap from program memory into EEPROM, it writes the ENTIRE array.  That's 2 bytes times the number of rows x columns in the matrix, times the number of layers.  Depending on the eeprom in question, this can cause an initial delay, since it is a large array.

Reply:
Gotcha; in practice I would never want to use VIA on STM32 without including #12567.    VIA program memory to EEPROM initialization seems to take a negligible amount of time (< 1 second) with 16 layers when including #12567

Reply:
> Gotcha; in practice I would never want to use VIA on STM32 without including #12567. VIA program memory to EEPROM initialization seems to take a negligible amount of time (< 1 second) with 16 layers when including #12567

Or, a fast eeprom chip. But I think most have write delays measured in ms, not us....